modules/example/src/main/scala/io/janstenpickle/trace4cats/example/Fs2AdvancedExample.scala

```diff
@@ -103,11 +102,10 @@ object Fs2AdvancedExample extends IOApp {
       }
 
   override def run(args: List[String]): IO[ExitCode] =
-    (for {
-      blocker <- Blocker[IO]
-      ep <- entryPoint[IO](blocker, TraceProcess("trace4catsFS2"))
-    } yield ep)
+    entryPoint[IO](TraceProcess("trace4catsFS2"))
       .use { ep =>
+        implicit val random: Random[Kleisli[IO, Span[IO], *]] = Random.javaUtilConcurrentThreadLocalRandom
```

Comment:
TODO: think on not passing it
switch to our own typeclass that has derviation via cats.effects.std.Random